.github/workflows/functional-containerinfra.yaml

```diff
@@ -26,12 +26,6 @@ jobs:
           - name: "victoria"
             openstack_version: "stable/victoria"
             ubuntu_version: "20.04"
-          - name: "ussuri"
-            openstack_version: "stable/ussuri"
-            ubuntu_version: "18.04"
-          - name: "train"
-            openstack_version: "stable/train"
```

Comment:
I'm not too familiar with this CI environment. But does this support using tags instead of branches? ie, could we use `train-eol` here instead of just removing it entirely?